### PR TITLE
Setup workflows

### PR DESCRIPTION
**Pull request by JaneSoo**
Set up example workflows to check python syntax and run test

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Python Package
+
+on:
+  pull_request:
+    types: [opened, edited]
```
Comment (lmilbaum) [inline .github/workflows/python-package.yml]: Not sure this is required. But, we do want to limit the workflow to PRs with `main` as the target branch

File: .github/workflows/python-package.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
```
Comment (lmilbaum) [inline .github/workflows/python-package.yml]: This whole section is not required

**Review by hubeadmin — COMMENTED** [review]
~Should we include some suggestions from @lioramilbaum? i.e. Isort/black etc.?~
^^ Not applicable

File: .github/workflows/python-package.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      fail-fast: false
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
```
Comment (lmilbaum) [inline .github/workflows/python-package.yml]: Please replace all those steps with one step which prints `Hello World`. This is a placeholder for triggering the A-Team pipeline

**Review by lmilbaum — CHANGES_REQUESTED** [review]
Please squash the commits and rename the pipeline. Lets name it - ateam

File: .github/workflows/python-package.yml (hunk 3 above)
Comment (hubeadmin) [reply .github/workflows/python-package.yml]: That makes more sense, ignore my previous comment :smiley:

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (hubeadmin) [reply .github/workflows/python-package.yml]: This is something along the lines of what we want:
```yml
on:
  # Trigger the workflow on push or pull request,
  # but only for the main branch
  pull_request:
    branches:
      - main
    types: [opened, edited]

```

File: .github/workflows/ateam.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    branches:
+      - main
+    types: [opened, edited]
+
+jobs:
+  print-msg:
+
+    runs-on: ubuntu-latest
+
+    steps:
```
Comment (lmilbaum) [inline .github/workflows/ateam.yml:14]: Where is the step which checks out the code?

File: .github/workflows/ateam.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+name: ateam
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [opened, edited]
+
+jobs:
+  print-msg:
```
Comment (lmilbaum) [inline .github/workflows/ateam.yml]: What is this key for?

Comment (lmilbaum) [reply .github/workflows/ateam.yml]: Shouldn't it be `build`?

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: .github/workflows/python-package.yml (hunk 3 above)
Comment (lmilbaum) [reply .github/workflows/python-package.yml]: The uses step should be kept

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**